collect_app/src/androidTest/java/org/odk/collect/android/feature/formmanagement/BulkFinalizationTest.kt

```diff
@@ -51,6 +56,35 @@ class BulkFinalizationTest {
             .assertNumberOfFinalizedForms(2)
     }
 
+    @Test
+    fun draftsAreBulkFinalizedInOldestFirstOrder() {
```

Comment:
I think we should actually test the send order (with auto send enabled) like you do in the other tests. The send order here is dependent on both the implementation of bulk finalization and auto send, so I think it's nice to check that both interact correctly to get the result we want.

Reply:
Here my goal wasn't to test the order of sending. I'm testing the order of finalizing forms. Wasn't that what you meant in the issue. Sending is tested in the two other tests.

Reply:
We really care about the order forms are sent, so the only reason to make bulk finalization happen in oldest to newest order is so that they get sent in the correct order when bulk finalizing.

Reply:
If it weren't for the order of sending, it wouldn't matter? I thought the order of finalizing might have something to do with entities, and that is also important.

Reply:
>  I thought the order of finalizing might have something to do with entities, and that is also important.

Oh that's a totally different thing I hadn't thought about. How about we take the bulk finalization stuff out of this PR and consider it separately.

Reply:
Ok I've reverted those commits. Do you want to have an issue for the order of finalizing?

Reply:
I'll make sure we have a discussion about it!